src/Util/ComposerAutoloaderFinder.php

```diff
@@ -19,45 +19,87 @@
  */
 class ComposerAutoloaderFinder
 {
+    private $rootNamespace;
+
     /**
      * @var ClassLoader|null
      */
     private $classLoader = null;
 
+    public function __construct(string $rootNamespace)
+    {
+        $this->rootNamespace = [
+            'psr0' => rtrim($rootNamespace, '\\'),
+            'psr4' => rtrim($rootNamespace, '\\').'\\',
+        ];
+    }
+
     public function getClassLoader(): ClassLoader
     {
         if (null === $this->classLoader) {
-            $this->classLoader = $this->findComposerClassLoader();
+            $this->findComposerClassLoader();
```

Comment:
Then we can change this back - I prefer to have all the other methods return a value, and then we set it here :)